pwndbg/vmmap.py

```diff
@@ -114,6 +116,16 @@ def clear_explored_pages():
     while explored_pages:
         explored_pages.pop()
 
+
+def add_custom_page(page):
+    custom_pages.append(page)
+    custom_pages.sort()
```

Comment:
Or just `bisect.insort(custom_pages, page)` :

```
In [2]: import bisect

In [3]: a = [1, 3, 4]

In [4]: bisect.insort(a, 2)

In [5]: a
Out[5]: [1, 2, 3, 4]

In [6]: bisect.insort(a, -1)

In [7]: a
Out[7]: [-1, 1, 2, 3, 4]
```

Reply:
Fixed. Thanks for suggesting `bisect`. I don't like the original one either.